File: C#/ProdusNou.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Tema_2_
{
    class ProdusNou : Produs
    {
        protected float ValoareEnergetica { get; set; }
        public string DataIntroducerii { get; set; }
        public void Afisare(int Stoc)
        {
            Console.WriteLine($"Produsul are in stoc {Stoc} bucati (Polimorfism static)\n");
            D.AfisareDetalii();
            Console.WriteLine();
        }
        public override void Afiseaza ()
        {
            Console.WriteLine("Produs din clasa <<ProdusNou>> (Polimorfism dinamic)\n");
            D.AfisareDetalii();
            Console.WriteLine();
        }
        public override void Afis()
        {
            Console.WriteLine("Produs din clasa <<ProdusNou>> (Interfata)\n");
        }
    }
}

File: C++/ProdusNou.cpp
#include "ProdusNou.h"
#include <iostream>
using namespace std;

void ProdusNou::afisare(int stoc) const
{
	cout << "Produsul are in stoc "<<stoc<<" bucati(Polimorfism static)"<< endl;
	d.afisareDetalii();
	cout << endl;
}
void ProdusNou::afiseaza() const
{
	cout << "Produs din clasa <<ProdusNou>>(Polimorfism dinamic)" << endl;
	d.afisareDetalii();
	cout << endl;
}
void ProdusNou::afis() const
{
	cout << "Produs de clasa ProdusNou (Interfata)" << endl;
	cout << endl;
}

/*void ProdusNou::set_vE(float e)
{
	valoareEnergetica = e;
}
float ProdusNou::get_vE()
{
	return valoareEnergetica;
}*/